build.gradle

```diff
@@ -350,3 +350,18 @@ class TestsReportTask extends DefaultTask {
         }
     }
 }
+
+// additional configuration that can't be done at settings.gradle
+gradleEnterprise {
+    buildScan {
+        // maintain a whitelist so that sensitive information (credentials) aren't accidentally published.
+        Set<String> whiteList = ['coverage', 'leak', 'blockhound', 'noLint',
```

Comment:
I also see `-Pretry=true` and `-PnoWeb` properties in github actions invocations in case they were missed 👍